jbehave-core/src/main/java/org/jbehave/core/embedder/PerformableTree.java

```diff
@@ -43,6 +40,7 @@
 import org.jbehave.core.steps.StepCreator.PendingStep;
 import org.jbehave.core.steps.StepResult;
 import org.jbehave.core.steps.Timer;
+import org.jbehave.core.steps.context.StepsContext;
```

Comment:
Do not reorganize imports without particular needs, as in future it will bring difficulties in merging/rebasing against original JBehave repo